### ISSUE TITLE
ZFS links lead to 404

### ISSUE THREAD

**Issue by jtagcat** [trapexit/backup-and-recovery-howtos#9]
Links reffed in the readme relating to ZFS don't lead anywhere, as they are not in the current main branch.

**Comment by trapexit** [comment]
Yeah. That is on purpose as the intent was to list out things and then write them up. As the repo didn't get much traction I've not expanded on it or found others to do so.
